torch/_prims/nvfuser_executor.py

```diff
@@ -20,11 +20,12 @@
 
 if torch.cuda.is_available():
     import nvfuser
-    from packaging.version import Version
+    from packaging.version import Version  # type: ignore[import]
```

Comment:
I'm not sure about this repo's way of importing `packaging` module. at glance, https://github.com/pytorch/pytorch/blob/d2b82feb41552e93efdd8f876b7a938d9657a1d6/test/onnx/test_verification.py#L14 is the only one which directly imports the module w/o guard. The others seem to import it from `pkg_resources` or https://github.com/pytorch/pytorch/pull/71902

Reply:
Yeah I think upstream relies on pkg_package in absence of `packaging`, tho I don't know what the difference is. I think I can use the lazy import from torch_version.py to handle this instead.
I'm not sure about the dependency between that with prim. We'll see if it breaks anything :crossed_fingers: 

Will ping you once I have it updated.

Reply:
switched to torch.torch_version.Version instead. Local testing seems to be working fine.